packages/react/toggle/src/Toggle.tsx

```diff
@@ -5,30 +5,32 @@ import { Primitive } from '@radix-ui/react-primitive';
 
 import type * as Polymorphic from '@radix-ui/react-polymorphic';
 
-const NAME = 'ToggleButton';
+const NAME = 'Toggle';
 const DEFAULT_TAG = 'button';
 
-type ToggleButtonOwnProps = Polymorphic.Merge<
+type ToggleOwnProps = Polymorphic.Merge<
   Polymorphic.OwnProps<typeof Primitive>,
   {
-    /** Whether the button is toggled or not, if controlled */
+    /**
+     * The controlled pressed state of the toggle.
```

Comment:
Just not sure about these renamed people don’t really need to know that it’s toggling `aria-pressed` under the hood. Personally I find pressed confusing anyway in aria because for me pressed is until I release.